lib/puppet/transaction/report.rb

```diff
@@ -345,9 +349,9 @@ def summary
 
       report[key].keys.sort { |a,b|
         # sort by label
-        if a == :total
+        if a == "total"
           1
-        elsif b == :total
+        elsif b == "total"
```

Comment:
@joshcooper I figured out why total wasn't being presented last in the summary

Reply:
These are stringified in `raw_summary`

Reply:
It'd be good to freeze those strings so we don't create extra object in memory